packs/feats/fledgling-flight.json

```diff
@@ -29,7 +29,13 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Ancestry Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "BaseSpeed",
+                "selector": "fly",
+                "value": "floor(floor(@actor.attributes.speed.total/2)/5)*5"
```

Comment:
```suggestion
                "value": "floor(@actor.attributes.speed.total/2)"
```

Reply:
The extra floor and multiplication is to round to the nearest value multiple to 5 - the later formula will round to 12 as the Flight speed if you have a 25 land speed.

The first formula ends results in 10.

Reply:
I don't see in the rules where movement is rounded down to increments of 5 - only that numbers are rounded down. Is there a reference for this?

Reply:
Went off of the Grid Movement [rules](https://2e.aonprd.com/Rules.aspx?ID=2355&Redirected=1#:~:text=Normally%2C%20movement%20costs%205%20feet,feet%20of%20movement%20per%20square.) on AoN.